Packs/ThreatIntelligenceManagement/Scripts/IncidentsPerFeed/IncidentsPerFeed.py

```diff
@@ -0,0 +1,114 @@
+import demistomock as demisto
+from CommonServerPython import *
+from typing import List, Dict, Union
+
+FEED = "feed"
+ACTIVE = "active"
+
+
+def get_incidents_count_by_feed(feed, from_date, query=None) -> int:
+    """Counts the incidents amount that fits the query and has indicators that came from the given feed.
+        Args:
+                feed: Feed from which the incidents indicator should be from
+                from_date: The date from the incidents should be queried
+                query: Additional filters for the 'getIncidents' command
+        @return:
+            total amount of incidents returned.
+    """
+    params = generate_query_params(feed, from_date, query)
+    res = demisto.executeCommand("getIncidents", params)
+    severe_incidents_count = res[0]["Contents"]["total"]
+    return severe_incidents_count
```

Comment:
The variable name `severe_incidents_count` seems inappropriate, as the default behavior of the function (unless a special query is specified) is to just return the total number of incidents. Please change the variable name accordingly.